docker-compose.yml

```
@@ -4,6 +4,10 @@ services:
     container_name: dynamodb-local
     ports:
       - "8000:8000"
+    command: "-jar DynamoDBLocal.jar -inMemory -sharedDb"
+    networks:
+      - bonded_default
+
 networks:
-  default:
-    driver: bridge
+  bonded_default:
+    driver: bridge
```

```suggestion
  default:
    driver: bridge

```

bondedは自動付与されるのでbonded_bonded_defaultになってしまう
defaultのみでOKだと思う